wordpress-mobile/WordPress-Android
# Create percentage bars in Stats cards

Fixes #9459
This PR adds the percentage bar to the Posts/Pages, Authors and Tags/Categories. The logic is that the biggest value (by views) gets 100% width and the other values get width by the percentage of the biggest value. I've created a drawable for the rounded corners of the progress bar that's used for showing the progress. I've also noticed the `subtext` field was never used so I removed it. 

To test:
* Go to Stats/DWMY
* Check that the Posts/Pages block contains the percentage bar
* Check that the lines with the bar are higher
* Check that the lines without the bar are still the same height
* Check that the Authors block contains the percentage bar
* Expand an Author
* Check that the author's posts don't have the percentage bar
* Check that the author's posts have lower height than authors
* Go to Insights/Tags&Categories
* Check that the items have the percentage bar
* Check that the expanded category items don't have the percentage bar

![Screenshot_1561012756](https://user-images.githubusercontent.com/1079756/59827199-52e98c80-9338-11e9-8807-85a0e984baa0.png)
![Screenshot_1561012748](https://user-images.githubusercontent.com/1079756/59827200-52e98c80-9338-11e9-815b-672edb283651.png)

Update release notes:
